chapters/it/_toctree.yml

```diff
@@ -39,3 +39,22 @@
   - local: chapter4/6
     title: Quiz di fine capitolo
     quiz: 4
+
+- title: 8. Come chiedere un aiuto
+  sections:
+  - local: chapter8/1
+    title: Introduzione
+  - local: chapter8/2
+    title: Cosa fare quando si riceve un errore
+  - local: chapter8/3
+    title: Chiedere aiuto sui forum
+  - local: chapter8/4
```

Comment:
Just FYI the reason the content doesn't build the preview yet is because the pipeline expects a translated file to exist for each entry in the table of contents. 

If you want to preview what you currently have, you could temporarily remove everything from here and below and then add them as you add the remaining files

Reply:
Yes I knew, sorry I didn't think about the Github Actions. The preview is working locally for me because I actually have all the files (the remaining are not translated and not committed) :)

I hope to finish soon 🤞

Reply:
Hey @EdAbati let me know if you need any help tidying up the table of contents :) The translated sections look great to me, so we can merge them once we've fixed the ToC

Reply:
Sorry for the delay @lewtun, lately I had less spare time than expected. This should be fixed now and I have also added section `4` and `4_tf`.

Let's see if I can squeeze the rest of the chapter in this PR (last minute before you review it)  😊 🤞